### PR TITLE
[#1905] Scrollbars only show now when needed and a scroll event takes place

### PR DESCRIPTION
**Pull request by mjschranz**
https://webmademovies.lighthouseapp.com/projects/65733/tickets/1905-allow-editor-scrollbars-to-be-only-visible-when-needed

### CODE REVIEW

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @secretrobotron received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/b850f3fa346367a/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/b850f3fa346367a/output.txt

Total script time: 0.80 mins

Server is running on http://pr857.butyr.org/templates/basic/
Run the Test Suite: http://pr857.butyr.org/test/

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @secretrobotron received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/77fda58975b6a4a/output.txt

File: src/util/scrollbars.js [hunk 1]
```diff
@@ -72,14 +75,21 @@ define( [ "core/eventmanager" ], function( EventManagerWrapper ){
       else {
         _handle.style.top = "0px";
       }
+      
+      // Toggles a fadding transition for our scrollbar
+      if ( !_element.classList.contains( "hidden" ) ) {
```
Comment (secretrobotron) [inline src/util/scrollbars.js:80]: I don't think you need this check? `hidden` won't appear in the class list twice if you call `add('hidden')` twice, right?

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/77fda58975b6a4a/output.txt

Total script time: 2.19 mins

**Pull request closed without merging**